services/tasks.js

```diff
@@ -83,7 +58,31 @@ const addTaskCreatedAtAndUpdatedAtFields = async () => {
   return operationStats;
 };
 
+const fetchOrphanedTasks = async () => {
+  try {
+    const userSnapshot = await usersQuery.fetchUsersNotInDiscordServer();
+    if (userSnapshot.empty) {
+      return [];
+    }
```

Comment:
left space for this.

Reply:
Will remove the brackets since it's unnecessary, didn't get what you meant by `left space for this`